src/element/textWysiwyg.test.tsx

```diff
@@ -813,5 +815,78 @@ describe("textWysiwyg", () => {
       ]);
       expect(text.containerId).toBe(null);
     });
+
+    it("should respect text alignment when resizing", async () => {
+      Keyboard.withModifierKeys({}, () => {
+        Keyboard.keyPress(KEYS.ENTER);
+      });
```

Comment:
no need for a `withModifierKeys` when no modifier is passed?

Reply:
right